collections-lists/LinkedList/LinkedList/CustomImplementation/CustomLinkedList.cs

```diff
@@ -0,0 +1,241 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LinkedList.CustomImplementation
+{
+    public class CustomLinkedList<T> : IEnumerable<Node<T>>
+    {
+        public Node<T>? First { get; set; }
+        public int Count { get; set; }
+
+        public void AddFirst(Node<T> nodeToAdd)
+        {
+            if (First is null)
+                First = nodeToAdd;
```

Comment:
Could we add brackets for this `if` statement? The article has already been changed.

Reply:
I have moved to the correct folder and added brackets to a couple of `if` statements that were missing.